client/my-sites/stats/stats-participation/index.tsx

```diff
@@ -0,0 +1,35 @@
+import config from '@automattic/calypso-config';
+import { useTranslate } from 'i18n-calypso';
+import DocumentHead from 'calypso/components/data/document-head';
+import JetpackColophon from 'calypso/components/jetpack-colophon';
+import Main from 'calypso/components/main';
+import { useSelector } from 'calypso/state';
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import PageViewTracker from '../stats-page-view-tracker';
+import StatsParticipationWizard from './stats-participation-wizard';
+
+const StatsParticipationPage = () => {
+	const translate = useTranslate();
+
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+	const isPurchaseEnabled = config.isEnabled( 'stats/subscribers-purchase-section' );
```

Comment:
```suggestion
	const isPurchaseEnabled = config.isEnabled( 'stats/paid-stats' );
```
Could we use another name for the flag please, as it's not subscribers.
Also, I'll use the flag for notices too. 😄 Thanks

Reply:
Of course, thanks for spotting it.